control-plane-operator/controllers/hostedcontrolplane/testdata/openshift-oauth-apiserver/zz_fixture_TestControlPlaneComponents.yaml

```diff
@@ -29,7 +29,7 @@ spec:
     metadata:
       annotations:
         cluster-autoscaler.kubernetes.io/safe-to-evict-local-volumes: work-logs
-        component.hypershift.openshift.io/config-hash: 19dc307e3415eb66
```

Comment:
Why is this changing?  does this mean this would cause a fleet wide rollout of this components?